DataFormats/PatCandidates/src/classes_def_objects.xml

```diff
@@ -149,7 +149,9 @@
    <version ClassVersion="10" checksum="2692173055"/>
   </class>
   <class name="std::vector<pat::tau::TauCaloSpecific>" />
-  <class name="pat::tau::TauPFEssential" ClassVersion="11">
+  <class name="pat::tau::TauPFEssential" ClassVersion="13">
```

Comment:
please remove ``` <version ClassVersion="12" checksum="853010979" /> ``` and rename the 13 to 12.
We should not introduce versions that were never and will never be in the release.